domain/crossmodelrelation/state/model/remoteapplication.go

```diff
@@ -529,7 +567,12 @@ func (st *State) insertRemoteApplicationOffererStatus(
 	sts status.StatusInfo[status.WorkloadStatusType],
 ) error {
 	insertQuery := `
-INSERT INTO application_remote_offerer_status (*) VALUES ($remoteApplicationStatus.*);
+INSERT INTO application_remote_offerer_status (*) VALUES ($remoteApplicationStatus.*)
```

Comment:
```
func (st *State) insertRemoteApplicationOffererStatus(
	ctx context.Context,
	tx *sqlair.TX,
	appID string,
	sts status.StatusInfo[status.WorkloadStatusType],
)
```
appID is a bad name for the function param
The db column we are using is the application_remote_offer uuid